SPECS/libguestfs/libguestfs.spec

```diff
@@ -8,301 +6,278 @@ Distribution:   Mariner
 
 # Architectures that we run the basic sanity-check test.
 #
-# The full test suite is done after the package has been built.  Here
-# we only do a sanity check that kernel/qemu/libvirt/appliance is not
-# broken.  To perform the full test suite, see instructions here:
-# https://www.redhat.com/archives/libguestfs/2015-September/msg00078.html
-#
 # Currently the basic sanity check is *broken* on:
 #
-# arm:     times out when running the test
 # aarch64: "MSI is not supported by interrupt controller" (RHBZ#1414081)
-# ppc64:   qemu doesn't work with TCG (RHBZ#1614948)
-# ppc64le: kernel doesn't boot on qemu (RHBZ#1435873)
-# s390x:   qemu TCG cannot emulate enough to boot the kernel
-#            (however KVM would work if it was available in Koji, so this
-#            is not a bug)
 %global test_arches x86_64
 
 # If there are patches which touch autotools files, set this to 1.
 %global patches_touch_autotools 1
 
-# The source directory.
-%global source_directory 1.44-stable
-
-# Filter perl provides.
-%{?perl_default_filter}
-
-# Unbreak the linker.
-%undefine _strict_symbol_defs_build
+%bcond_without appliances
 
-%bcond_without applicances
 %bcond_with php
 %bcond_with inspect-icons
+%bcond_with man-pages
 
-Summary:       Access and modify virtual machine disk images
-Name:          libguestfs
-Version:       1.44.0
-Release:       5%{?dist}
-License:       LGPLv2+
-
+Summary:        Access and modify virtual machine disk images
+Name:           libguestfs
+Version:        1.44.0
+Release:        6%{?dist}
+License:        LGPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
 # Source and patches.
-URL:           http://libguestfs.org/
-Source0:       http://libguestfs.org/download/%{source_directory}/%{name}-%{version}.tar.gz
-
+URL:            https://libguestfs.org/
+Source0:        https://libguestfs.org/download/1.44-stable/%{name}-%{version}.tar.gz
```

Comment:
Nit: would be nice to use macros to extract the `maj.min` version for use here

Reply:
I thought about that but decided against it because... it was late and I couldn't mentally take another small modification.;) I'll do this now.